src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -50,6 +59,13 @@ public String getUrlForWiki(String wikiName) {
     return composeUrl(prefix, wikiName, suffix);
   }
 
+  public String getUrlForWikiWithWWW(String wikiName) {
```

Comment:
is this method returning url that contains www ?
or the path provided in parameter should contain www ?
the method name suggest the second option, but the returned value suggest the first option
change the method name ti